.github/workflows/mmteb.yml

```diff
@@ -26,21 +46,13 @@ jobs:
           make install
           pip install tabulate # for the table creation
 
-
-      - name: Validate jsonl points files
-        run: python docs/mmteb/validate_points.py
-
       - name: Create table
         run: python docs/mmteb/create_points_table.py
-
-        
+    
       - name: Push table
-        # only run on push to main
-        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
         run: |
           git config --global user.email "github-actions[bot]@users.noreply.github.com"
           git config --global user.name "github-actions[bot]"
-          git add docs/mmteb/points_table.md
           # check if the branch is up to date with the remote if not push the changes
-          git diff --exit-code || git commit -m "Update points table" && git push
+          git diff --exit-code || git add docs/mmteb/points_table.md && git commit -m "Update points table" && git push
```

Comment:
I believe you [need a token](https://stackoverflow.com/questions/57921401/push-to-origin-from-github-action) to be allowed to push to main (I might be wrong)

Reply:
Ya good call, we do. The job uses the same token as before (line#37), and it was working for at least commit 1c17394f6147acee175cdbb4dfaeb9a49ac96200.

PRs from this morning didn't trigger a commit to the table page was due to `git diff`, and likely not related to the token.

Reply:
Ahh right that is, because the GH changelog did not show you added it (it just as an edit of the previous). This seems like then!